docs/Manual/03 Schedules.md

```diff
@@ -104,7 +106,7 @@ ii = schedule.split(i,4)
 iii = schedule.split(i,2)
 iiii = schedule.split(ii,2)
 ```
-After the first split, the iteration space has a shape of (4, 4, 10, 11). After the second split, the shape becomes (2, 2, 4, 10, 11). Finally, the shape becomes (2, 2, 2, 2, 10, 11). The transformed schedule corresponds to the following python code:
+After the first split, the iteration space has the shape (4, 4, 10, 11). After the second split, the shape becomes (2, 2, 4, 10, 11). Finally, the shape becomes (2, 2, 2, 2, 10, 11) after the third split. The transformed schedule corresponds to the following python code:
```

Comment:
"the following Python code" (we should capitalize Python in the text to be consistent)